Fluid/Filters/StringFilters.cs

```diff
@@ -299,7 +299,10 @@ public static ValueTask<FluidValue> TruncateWords(FluidValue input, FilterArgume
                 source = "";
             }
 
-            source += ellipsis;
+            if (size <= words)
```

Comment:
Shouldn't this be done also in the truncate filter?

Reply:
Nope